lib/renderer/remote.ts

```diff
@@ -1,8 +1,6 @@
-'use strict'
+import { remote, Remote } from 'electron'
 
-const { remote } = require('electron')
-
-exports.getRemote = function (name) {
+export function getRemote (name: keyof Remote) {
```

Comment:
Just as a point of standardization I've been using `Electron.Remote` instead of importing the `Remote` interface elsewhere in code.

Is there a strong case for one over the other?  (I'm not set on either)

Reply:
Same end result, so it doesn't really matter. Let me go with your preferred version!